website/docs/r/nsxt_network_imported.html.markdown

```diff
@@ -88,12 +121,13 @@ Static IP Pools  support the following attributes:
 * `end_address` - (Required) The final address in the IP Range
 
 ## Attribute Reference
-* `nsxt_logical_switch_id` - ID of an existing NSX-T segment
+* `nsxt_logical_switch_id` - ID of NSX-T logical switch used by this network
+* `dvpg_id` - ID of Distributed Virtual Port Group used by this network
```

Comment:
Should we mention that it comes from vSphere here?

Reply:
This is tricky to state as most probably the ID would not exactly match vSphere ID as we have seen already